addon/serializers/linked-node.js

```diff
@@ -0,0 +1,14 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serializeIntoHash(hash, typeClass, snapshot, options) {
+        if (options.forRelationship) {
+            hash.data = [{
```

Comment:
Does this override any preexisting `data` property? Do we mean to add to the existing properties instead?